File: testers/avro/java-async/generated-and-checked-in/com/example/events/consumer/LinkedListNodeHandler.java
package com.example.events.consumer;

import com.example.events.LinkedListNode;
import com.example.events.header.StandardHeaders;
import java.util.concurrent.CompletableFuture;

/** Handler interface for linked-list-node topic events */
public interface LinkedListNodeHandler {
  /** Handle a message from the topic */
  CompletableFuture<Void> handle(String key, LinkedListNode value, StandardHeaders headers);
}

File: testers/avro/kotlin/generated-and-checked-in/com/example/events/consumer/LinkedListNodeHandler.kt
package com.example.events.consumer

import com.example.events.LinkedListNode
import com.example.events.header.StandardHeaders

/** Handler interface for linked-list-node topic events */
interface LinkedListNodeHandler {
  /** Handle a message from the topic */
  abstract fun handle(
    key: kotlin.String,
    value: LinkedListNode,
    headers: StandardHeaders
  )
}